### PR TITLE
Fixed generic entity row for climate entities

### PR DESCRIPTION
**Pull request by kukulich**
## Proposed change

It fixes entity row for climate entities. It partly reverts changes made in https://github.com/home-assistant/frontend/pull/8028
The point "Since the value can with out problem use up two lines" is not true for climate entities.

Screenshot are for my real entities and in Czech translation - czech words are longer than english.

Before:
before

After:
after

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by kukulich** [comment]
@spacegaier I would be happy if you can find better solution.

File: src/panels/lovelace/components/hui-generic-entity-row.ts [hunk 1]
```diff
@@ -165,7 +165,7 @@ class HuiGenericEntityRow extends LitElement {
       .info {
         margin-left: 16px;
         margin-right: 8px;
-        flex: 1 0 30%;
+        flex: 1 0 60px;
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-generic-entity-row.ts:168]: Why don't we allow it to shrink?

**Comment by bramkragten** [comment]
This is also an issue for the media player row:
image
image

File: src/panels/lovelace/components/hui-generic-entity-row.ts (hunk 1 above)
Comment (kukulich) [reply src/panels/lovelace/components/hui-generic-entity-row.ts]: @bramkragten Shrink can help too so forcepushed with "shrink".

**Comment by bramkragten** [comment]
@spacegaier could you check if this causes issues with your previous fix?

**Comment by kukulich** [comment]
@spacegaier Could you please check this?

**Comment by spacegaier** [comment]
It is on my list for the weekend. I need to look into the original raised issue and my resulting PR to remind myself what and why was done...

**Comment by spacegaier** [comment]
Just as a note: I have not forgotten it, but just did not find the time so far. This weekend though 🤞 .

**Comment by kukulich** [comment]
@bramkragten @spacegaier Is it possible to make it to 2021.4.0, please? :) It's broken since 2021.1.0... The fix may not be perfect but it should be better than actual state. The bug caused problems and reports in other repositories, see mentioned issues.

**Comment by spacegaier** [comment]
Fine with me. Of course there still can be situations where this does not solve it:

image

And it means that the name of the entity is sometimes hidden e.g. only "Te" visible here:

image

And mainly: It re-introduces the other issue from before that we cut off some text earlier:

With this PR:
image

Without this PR:
image

**Review by spacegaier — APPROVED** [review]

**Pull request merged**